math/matrix/inc/TVectorT.h

```diff
@@ -27,15 +27,15 @@
 template<class Element> class TVectorT : public TObject {
 
 protected:
-   Int_t    fNrows;                // number of rows
-   Int_t    fRowLwb;               // lower bound of the row index
-   Element *fElements;             //[fNrows] elements themselves
+   Int_t    fNrows{0};                // number of rows
```

Comment:
Is there a reason to prefer direct initialization over copy initialization on scalar types? The direct initialization disables type narrowing and it is more widely used for objects.

I'd change the pattern for scalar default init to `Int_t fNrows = 0`. This way the code would look more 'standard'.